eyebrowse.el

```diff
@@ -607,7 +610,9 @@ will be set up with `eyebrowse-setup-evil-keys' as well."
          (separator (propertize eyebrowse-mode-line-separator
                                 'face 'eyebrowse-mode-line-separator))
          (current-slot (eyebrowse--get 'current-slot))
-         (window-configs (eyebrowse--get 'window-configs)))
+         (window-configs (if (eq eyebrowse-mode-line-style 'current)
+                             `(,(assq current-slot (eyebrowse--get 'window-configs)))
```

Comment:
No need for unquote here, just use `(list (assq ...))`.

Reply:
Also, `assq` is wrong because numbers being `eq` to each other isn't something you can rely on (the bignum change could very well break this assumption. Use `assoc`.